docker-compose/docker-compose-certify/docker-compose.yml

```diff
@@ -48,17 +76,20 @@ services:
     environment:
       - artifactory_url_env=http://artifactory-server:8080/
       - container_user=mosip
-      - active_profile_env=default
+      - active_profile_env=default,mock-identity
```

Comment:
mock profile has a dependency on 1.4.0 for redis cache sharing

Reply:
But 1.4.1 should be compatible with redis cache

Reply:
right now the plugins are using esignet 1.4.0 DTO's due to the serialization issue with redis cache, we have to change in mock and mosip id plugins if we have to make changes here since we are pulling the plugins from artifactory 
reference: https://github.com/mosip/digital-credential-plugins/blob/a96fada5b8eefa00282cadab1c698f429223c0b3/mock-certify-plugin/pom.xml#L75

Reply:
added this as a note in read me